components/NewOutivityForm/index.js

```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function NewOutivityForm({
+  CreateOutivity,
+  handleCancel,
+  selectedImage,
+}) {
+  return (
+    <main>
+      <StyledNewOutivitiesForm onSubmit={CreateOutivity}>
+        <h1>New Outivity</h1>
+        <StyledNewOutivitiesFormFields>
+          <StyledNewOutivitiesFormField>
+            <label htmlFor="image">Image</label>
```

Comment:
with which form element is this label connected?

if you want to connect it with the outivityImage, the htmlFor of the label needs to match the id of the form element.

```suggestion
            <label htmlFor="outivityImage">Image</label>
```